main.css

```diff
@@ -0,0 +1,115 @@
+:root {
+    --pink: #f7e1d7ff;
+    --main: rgb(255, 233, 210);
+    --green: #b0c4b1ff;
+    --pretty: rgb(121, 159, 117);
+    background-color: lightblue;
+  }
+  
+  html {
+    height: 100%;
+  }
+  
+  body {
+    height: 100%;
+    background-color: var(--main);
+  }
```

Comment:
If you want to remove the blue border (you certainly don't have to though!), add`margin: 0` to this set of rules